src/modules/addEditRemove.js

```diff
@@ -1,35 +1,56 @@
 // Function to save to local storage
-export const storeItem = (todo) => {
-  localStorage.setItem('items', JSON.stringify(todo));
+export const storeItem = (todo, localStorage) => {
+  if (localStorage) {
+    localStorage.setItem('items', JSON.stringify(todo));
+    console.log('Stored items:', localStorage.getItem('items'));
+  }
 };
 
-// Function to save to local storage
-export const getItem = () => JSON.parse(localStorage.getItem('items')) || [];
+// Function to retrieve from local storage
+export const getItem = (localStorage) => {
+  if (localStorage) {
+    const storedItems = localStorage.getItem('items');
+    console.log('Retrieved items:', storedItems);
```

Comment:
You are doing well so far apart from the following issues
#8 Kindly avoid leaving the debugging codes in your js files this will help optimize your code.